Celeste.Mod.mm/Patches/LevelLoader.cs

```diff
@@ -190,8 +203,38 @@ public patch_LevelLoader(Session session, Vector2? startPosition = default)
                 LoadingThread();
             } catch (Exception e) {
                 string sid = session?.Area.GetSID() ?? "NULL";
-                patch_LevelEnter.ErrorMessage = Dialog.Get("postcard_levelloadfailed").Replace("((sid))", sid);
-                Logger.Log(LogLevel.Warn, "LevelLoader", $"Failed loading map {sid}");
+                if (patch_LevelEnter.ErrorMessage == null) {
+                    if (e is KeyNotFoundException && e.StackTrace.Contains("TileHandler") && e.Data.Count == 3) {
```

Comment:
Here we could just add a `Source` tag or something to `Data` instead of checking the StackTrace, or if we set the error message, we could base the check off that.